app/assets/stylesheets/components/autocomplete.scss

```diff
@@ -2,6 +2,11 @@
 @import '@reach/combobox/styles';
 
 .crayons-autocomplete {
+  // Override of the default hidden class to make sure it takes priority over any display style attached to the 'replaceElement' textarea
+  .hidden {
+    display: none !important;
```

Comment:
Are the styles you're trying to overwrite on the elements themselves? If not can we just make the rule more specific? It just feels a shame to use important 😅

Reply:
I know what you mean it always feels so wrong to use `important` 😄 

In the current use case in comments, I think we could actually omit the `important`, but I'm adding it as future-proofing. The way this autocomplete component works, is we copy all styles from some other element (e.g. the initially loaded comment box), and apply them to two inputs, one of which needs to be hidden at all times. The original element is then removed completely.

I feel like in this case, `important` is a good fit - it makes sure that no matter how specific a style comes from the original element, we'll definitely never have a situation where one of those hidden elements is visible.

It's a bit like in our `reset.scss` where we make sure animations are removed for users with `prefers-reduced-motion`, in that it's a situation where regardless of any previous style, wherever this component is used, we want to override it.